README.md

```diff
@@ -105,139 +109,51 @@ query service for raw-data only. Query for rolledup data is not built yet.
+- Generate some test data into Kafka by running the `test/data-generator` module [in the test directory of the bundle](https://github.com/racker/ceres-bundle/tree/master/test) by using `mvn spring-boot-:run`
+- In the [apps directory of the bundle](https://github.com/racker/ceres-bundle/tree/master/apps) go into the `ingestion-service` and run it using `mvn spring-boot:run`.
 
 Now, you should have some data to play with.
-- On terminal window, run command `docker exec -it influxdb influx` to get to `influx-cli`
+- On terminal window, run command `docker exec -it influxdb influx` to access the `influx-cli`
 - Run `use db_0` to use database `db-0`
 - Run `select * from MAAS_cpu limit 10` to get some records, so that we can get `tenantId` to query
 - When you have some results showing up for the query, look for the column `tenantId` and 
-grab one `tenantId` from the rows. For example: `ENCORE-MAAS-account-name-0`
+grab one `tenantId` from the rows. For example: `MAAS-account-name-0`
 
-Now, you can run `query-service` and use any REST client (for example, Insomnia) to play
-around with the service.
-- Open REST client `Insomnia` (or REST client tool of your choice)
-    - use HTTP method `GET`
-    - url example: `http://localhost:8080/intelligence-format-query?db=ENCORE-MAAS-account-name-0&q=select * from MAAS_cpu where time >= now()-8h`
-  
-  Some notes on testing/developing with the security features enabled:
-    - When using Insomnia/curl/other local methods of testing, do not use loopback (localhost/127.0.0.1). Use your network IP address. (`ifconfig -a`)
-    Make sure that network ip address is covered with the whitelisted ip address range in the development profile.
-    - When calling grafana/admin style queries (multitenant) use port 8080 (where the query service runs)
-    - When calling intelligence style queries (single tenant) use port 8180 (repose). 
-    Repose (run ceres-test-infrastructure for repose) will provide token-based security.
-    Use curl to get your Rackspace test token.
-    ```
-  curl https://identity.api.rackspacecloud.com/v2.0/tokens  \
-      -X POST \
-      -d '{"auth":{"passwordCredentials":{"username":"mypersonaltestaccount","password":"mypassword"}}}' \
-      -H "Content-type: application/json" | python -m json.tool
-    ```
+When accessing the `/v1.0/tenant` APIs in a local development environment, you can simulate the behavior of Repose or make the invocation via Repose.
+
+To simulate the behavior of Repose, access the query service directly at port 8080 and pass the
+headers:
+- `X-Tenant-Id`
+- `X-Roles`, where development and test profiles default to allow "dev:default"
+
+To access the tenant APIs via Repose, ensure the `repose` service is started from the `test/instructure` bundle module and access the API via port 8180. The following header needs to be set:
+- `X-Auth-Token`: a valid Identity token
```

Comment:
I am getting 403 on this request now http://localhost:8180/v1.0/tenant/1048681/intelligence-format-query/measurements

Reply:
works fine now @itzg !